packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -44,6 +45,7 @@ export async function getDestination(
 ): Promise<Destination | null> {
   return (
     searchEnvVariablesForDestination(options) ||
+    searchRegisteredDestination(options) ||
```

Comment:
Now, this list/order grows, which needs to be documented so the priority is clear when reaching conflics.

Reply:
I added a test, and yes the docs need to be adjusted to include the new `registerDestination` method and also make the line clear when to use what.

Reply:
General question: I think this makes sense for explicitness, but wouldn't it have sufficed to `getDestinationFromDestinationService`?

Reply:
no becaue the registeredDestination uses a different cache without expiration at the moment. Since the current implementation did not allow for entry specific life time

Reply:
Perhaps this is not true and you could set it with undefined and it will live longer than the default value but than I would say this is a bug